bin/cron/birthday.send.php

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/php5 -q
+<?php
+/***************************************************************************
+ *  Copyright (C) 2003-2015 Polytechnique.org                              *
+ *  http://opensource.polytechnique.org/                                   *
+ *                                                                         *
+ *  This program is free software; you can redistribute it and/or modify   *
+ *  it under the terms of the GNU General Public License as published by   *
+ *  the Free Software Foundation; either version 2 of the License, or      *
+ *  (at your option) any later version.                                    *
+ *                                                                         *
+ *  This program is distributed in the hope that it will be useful,        *
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of         *
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the          *
+ *  GNU General Public License for more details.                           *
+ *                                                                         *
+ *  You should have received a copy of the GNU General Public License      *
+ *  along with this program; if not, write to the Free Software            *
+ *  Foundation, Inc.,                                                      *
+ *  59 Temple Place, Suite 330, Boston, MA  02111-1307  USA                *
+ ***************************************************************************/
+
+require_once 'connect.db.inc.php';
+require_once 'plmailer.php';
+ini_set('memory_limit', '128M');
+
+$pids = XDB::query('SELECT  pid
+                      FROM  profiles
+                     WHERE  birthday BETWEEN CURDATE() AND DATE_ADD(CURDATE(), INTERVAL 1 DAY)
+                  ORDER BY  pid');
+$iterator = Profile::iterOverPIDs($pids->fetchColumn());
```

Comment:
This can be done with a ``ProfileFilter``:
```php
$pf = new ProfileFilter(new PFC_And(
    new UFC_Birthday('=', time()),
    new UFC_HasValidEmail(),
    new PFC_Not(new UFC_Dead()),
));
$iterator = $pf->iterProfiles();
```